extensions-core/stats/src/main/java/org/apache/druid/query/aggregation/variance/VarianceAggregatorCollector.java

```diff
@@ -60,11 +60,11 @@ public static VarianceAggregatorCollector from(ByteBuffer buffer)
   }
 
   public static final Comparator<VarianceAggregatorCollector> COMPARATOR = (o1, o2) -> {
-    int compare = Longs.compare(o1.count, o2.count);
+    int compare = Doubles.compare(o1.nvariance, o2.nvariance);
```

Comment:
Will this sort the same way as `getVariance(isVariancePop)` in `finalizeComputation`?

Reply:
I think so. I based this on the comment against the `nvariance` variable

```
double nvariance; // sum[x-avg^2] (this is actually n times of the variance)
```

`getVariance(isVariancePop)` looks like it divides by the count to get variance instead of nvariance.